hw/ip/otp_ctrl/data/dif_otp_ctrl.h.tpl

```diff
@@ -288,6 +288,34 @@ dif_result_t dif_otp_ctrl_check_integrity(const dif_otp_ctrl_t *otp);
 OT_WARN_UNUSED_RESULT
 dif_result_t dif_otp_ctrl_check_consistency(const dif_otp_ctrl_t *otp);
 
+/**
+ * Locks out access to the direct access interface registers.
+ *
+ * This function is reentrant: calling it while functionality is locked will
+ * have no effect and return `kDifOtpCtrlOk`.
```

Comment:
nit: It's just `kDifOk` :)

Reply:
Lol it looks like all function descriptions have this error. I will correct that.